pydantic_redis/abstract.py

```diff
@@ -21,6 +21,9 @@ class Config:
         arbitrary_types_allowed = True
         orm_mode = True
 
+def pydantic_serializer(data):
```

Comment:
Rename to `fallback_serializer` as it will house any serialization that orjson cannot handle.

Reply:
```suggestion
def fallback_serializer(data):
```